templates/templates/docs/shared/_docs.html

```diff
@@ -98,6 +98,30 @@ <h3 class="p-side-navigation__heading">{{ nav_group.navlink_text }}</h3>
     </aside>
 
     <main class="col-9">
+      {% if document.headings_map %}
+        <p><strong>Contents:</strong></p>
```

Comment:
I think maybe we should remove this line and just put the table without a heading? What do you think?

Reply:
I agree. I will add an aria-label instead for screen readers